src/azul/queues.py

```diff
@@ -108,6 +100,27 @@ def _dump_messages(self, messages, queue_url, path):
             json.dump(content, file, indent=4)
         logger.info('Wrote %i messages', len(messages))
 
+    def _get_messages(self, queue):
+        messages = []
+        while True:
+            message_batch = queue.receive_messages(AttributeNames=['All'],
+                                                   MaxNumberOfMessages=10,
+                                                   VisibilityTimeout=300)
+            if not message_batch:  # Nothing left in queue
+                break
```

Comment:
```suggestion
                return messages
```

Reply:
was trying to minimize the changes in this file, ill refactor it.